plugins/cisco_asa_logs.yaml

```diff
@@ -3,9 +3,11 @@ title: Cisco ASA
 description: Log parser for Cisco ASA
 parameters:
   - name: listen_port
+    description: A port which the agent will listen for syslog messages
     type: int
     default: 5140
   - name: listen_ip
+    description: A syslog ip address of the form `<ip>`
```

Comment:
I think we can drop the `of the form <ip>` part? Not sure it adds much help to the user. Probably the same in every plugin that has this.

Reply:
Sure, went through and took that part out